artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/ha/ReplicatedPolicy.java

```diff
@@ -158,6 +164,7 @@ public ReplicaPolicy getReplicaPolicy() {
          if (groupName != null && groupName.length() > 0) {
             replicaPolicy.setGroupName(groupName);
          }
+         replicaPolicy.setQuorumVoteWait(quorumVoteWait);
```

Comment:
pass the parameter using the `ReplicePolicy` constructor to not escape the validation that it could perform